spec/models/book_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  subject(:book) do
+    Book.new(
```

Comment:
Let's move this to a factory! Please, use [factory_bot](https://github.com/thoughtbot/factory_bot) as suggested in the card!